### DISCUSSION TITLE
Using two different displays for one ESP32

### DISCUSSION THREAD

**Discussion by griffinmt** [Bodmer/TFT_eSPI#2524] [General]
This subject has probably been addressed before, but as an old-timer, I cant remember. I am making an application that runs on a single esp32 but uses two different displays. One is round using a GC9A01 driver with sprites, and the other is an ILI9341 display. How do I configure things to make this work. I had thought I might have to clone the TFT_eSPI library under a different name, but not sure of the implications. Can you help me out someway PLEASE.
TIA
Martyn

**Comment by Bodmer** [comment]
That is not supported so you would need to create a copy and search/replace all the strings TFT_eSPI in the entire library.

**Reply by griffinmt** [reply]
Thanks for your answer. That seems like a lot to do with lots of potential errors on my part. Oh well!!

**Comment by Bodmer** [comment]
Processors are typically cheaper than the displays, so in this situation I would use one processor per display and link them for data exchange.

**Comment by griffinmt** [comment]
I have been trying to link three of them. Tried bluetooth but unable to
pair them in a three-way. Would consider using webSocketsClient and Server,
but confused how each can send to the others when ever they need.
